frontend/src/styles/utilities.stories.tsx

```diff
@@ -66,14 +66,14 @@ export const SpaceAndGap = (): JSX.Element => {
                 <LemonButton type="primary">Button</LemonButton>
                 <LemonButton type="primary">Button</LemonButton>
                 <LemonButton type="primary">Button</LemonButton>
-                <code>gap-4 (1rem or 16px)</code>
+                <code>gap-4 (1rem or 10px)</code>
             </div>
 
             <div className="flex items-center gap-8">
                 <LemonButton type="primary">Button</LemonButton>
                 <LemonButton type="primary">Button</LemonButton>
                 <LemonButton type="primary">Button</LemonButton>
-                <code>gap-8 (2rem or 32px)</code>
```

Comment:
1 rem is 16 px in all browsers by default (though user-configurable for accessibility)

Reply:
Oh this was me using CMD+F and replace too easily, good catch 😅